index.bs

```diff
@@ -925,32 +981,34 @@ return a new {{Promise}} |promise| and run the following steps in
 parallel:
 
   1. Let |device| be the target {{USBDevice}} object.
-  2. If |device| is no longer connected to the system, <a>reject</a> |promise|
+  1. If |device| is no longer connected to the system, <a>reject</a> |promise|
      with a {{NotFoundError}} and abort these steps.
-  3. If <code>|device|.{{USBDevice/opened}}</code> is not equal to
+  1. If <code>|device|.{{USBDevice/opened}}</code> is not equal to
      <code>true</code> <a>reject</a> |promise| with an {{InvalidStateError}}
      and abort these steps.
-  4. <a>Check the validity of the control transfer parameters</a> and abort
+  1. <a>Check the validity of the control transfer parameters</a> and abort
      these steps if |promise| is rejected.
-  5. If |length| is greater than the <code>wMaxPacketSize0</code> field of the
+  1. If |length| is greater than the <code>wMaxPacketSize0</code> field of the
      device's <a>device descriptor</a>, <a>reject</a> |promise| with a
      {{TypeError}} and abort these steps.
-  6. Let |result| be a new {{USBInTransferResult}} and let |buffer| be a new
-     {{ArrayBuffer}} of |length| bytes.
-  7. Issue a control transfer with the setup packet parameters provided in
-     |setup| and the data transfer direction in <code>bmRequestType</code> set
+  1. Let |buffer| be a host buffer with space for |length| bytes.
+  1. Issue a control transfer with the setup packet parameters provided in
```

Comment:
I don't see where |buffer| is passed to the control transfer. How does the other system know to write there?